backend/src/main/resources/application.properties

```diff
@@ -1,7 +1,8 @@
-server.port=8081
+server.port=8082
```

Comment:
Ha megváltozik a server port, akkor a Dockerfile-ban és a compose-ban is utána kellene húzni

Reply:
Kiszaladt a local change.

Reply:
De ha nincs különösebb oka, akkor ne változtassuk meg.